README.md

```diff
@@ -48,7 +50,7 @@ If you have Jupyter installed, you can create a notebook using IRkernel from the
 You can also start other interfaces with an R kernel:
 
 ```bash
-# “ir” is the kernel name installed by the above 'IRkernel::installspec()'
```

Comment:
what terminal emulator do you use? cmd, cmder or powershell? what git distribution? what locale?

one of them is buggy and doesn’t support unicode, as the commit message is obviously fine: a27c12e51b24e87f91127bd11f668934bb211b7a